pkg/util/envvar/envvars.go

```diff
@@ -0,0 +1,28 @@
+package envvar
+
+import (
+	"sort"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+func Merge(existing, desired []corev1.EnvVar) []corev1.EnvVar {
```

Comment:
do we want to call it MergeWithOverride to be explicit?

Reply:
good idea 👍